test/fixed.jl

```diff
@@ -275,8 +275,8 @@ end
         xs = typemin(F):eps(F):typemax(F)
         fneg(x) = -float(x)
         @test all(x -> wrapping_neg(wrapping_neg(x)) === x, xs)
-        @test all(x -> saturating_neg(x) == clamp(fneg(x), F), xs)
-        @test all(x -> !(typemin(F) < fneg(x) < typemax(F)) ||
+        @test all(x -> saturating_neg(x) === clamp(fneg(x), F), xs)
+        @test all(x -> !(typemin(F) <= fneg(x) <= typemax(F)) ||
```

Comment:
I had excluded `typemin` and `typemax` with the intention of writing generic code for 100% testing. They have a risk of causing inconsistencies due to the rounding errors in floating point operations.
However, I decided against supporting non-8-bit types in 100% tests for multiplication. So we can safely include `typemin` and `typemax` for 8-bit types.